packages/ui/package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "@repo/ui",
   "version": "6.0.0",
+  "private": true,
```

Comment:
Currently our packages-release github action is trying to publish this (causing errors)